### ISSUE TITLE
git push error

### ISSUE THREAD

**Issue by dorothykiz1** [python-sprints/pandas-mentoring#4]
am getting this error when I try to push kindly help

remote: Permission to python-sprints/pandas-mentoring.git denied to dorothykiz1.
fatal: unable to access 'https://github.com/python-sprints/pandas-mentoring.git/': The requested URL returned error: 403

**Comment by datapythonista** [comment]
Good job on reaching that point @dorothykiz1, that was expected, the workflow you need to follow is counter-intuitive in that part.

@martinagvilas, @sparalic, @galuhsahid do you mind helping @dorothykiz1 with this?

**Comment by martinagvilas** [comment]
Hi @dorothykiz1, did you fork the pandas-mentoring repository into your own github account? This might be the issue. You should fork the repository, push the changes you made first to your fork, and then create a pull request from there. See https://stackoverflow.com/questions/17857283/permission-denied-error-on-github-push for an overview of the steps.

**Comment by dorothykiz1** [comment]
Thank you @martinagvilas this could be possibly my problem had not forked thank you let me try again

**Comment by datapythonista** [comment]
@dorothykiz1 if you resolved this problem (I think you did), feel free to close this issue.

**Comment by dorothykiz1** [comment]
Sure
